assets/src/edit-story/components/header/buttons/publish.js

```diff
@@ -59,7 +66,7 @@ function Publish() {
 
   useEffect(() => {
     if (showDialog) {
-      trackEvent('missing_title_dialog');
+      trackEvent('review_prepublish_checklist');
```

Comment:
@swissspidy is there anything to do when updating a tracking event? I couldn't find anything else that seems to need an updated.